app/assets/stylesheets/_buttons.scss

```diff
@@ -50,7 +50,7 @@
 @mixin transition-all($time) {
     -webkit-transition: all $time ease;
     -moz-transition: all $time ease;
-    o-transition: all $time ease;
+    -o-transition: all $time ease;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Avoid vendor prefixes.